src/app/modules/main/module.nim

```diff
@@ -333,6 +333,12 @@ proc createCommunitySectionItem[T](self: Module[T], communityDetails: CommunityD
     # We will update the model later when we finish loading the accounts
     self.controller.asyncGetRevealedAccountsForAllMembers(communityDetails.id)
 
+    # If there tokens are already in the model, we should keep the existing community tokens, until
+    # getCommunityTokensDetailsAsync will trigger onCommunityTokensDetailsLoaded
```

Comment:
Alternatively, I can call `createTokenItemImproved` inplace because we already have a CommunityTokenDto. But I'm not sure if this is possible without communityTokenJsonItems.